pkg/pillar/types/errortime.go

```diff
@@ -74,6 +162,26 @@ func (etsPtr *ErrorAndTimeWithSource) SetError(errStr string, errTime time.Time)
 	etsPtr.Error = errStr
 	etsPtr.ErrorSourceType = ""
 	etsPtr.ErrorTime = errTime
+	etsPtr.ErrorSeverity = ErrorSeverityError
+	etsPtr.ErrorEntities = []*ErrorEntity{}
+	etsPtr.ErrorRetryCondition = ""
```

Comment:
Can't this call just set ErrorSourceType = "" and the call a common SetErrorWithDescription() function so we have the same code?

Reply:
modified